plugins/devtools-backend/src/api/DevToolsBackendApi.ts

```diff
@@ -203,17 +203,21 @@ export class DevToolsBackendApi {
     return configInfo;
   }
 
-  public async listInfo(): Promise<DevToolsInfo> {
-    const operatingSystem = `${os.hostname()}: ${os.type} ${os.release} - ${
-      os.platform
-    }/${os.arch}`;
+  public async listResourceUtilization(): Promise<string> {
```

Comment:
I feel it's a bit unnecessary to add this extra method tbh, the `listInfo` one is already fairly cheap, and with a tiny bit of caching of the static values there's really not much difference between the two at all. I'm thinking it's better to just call `listInfo` instead and then optimize it a bit if you want.